realm/realm-library/src/androidTest/java/io/realm/entities/RealmRecyclerAdapter.java

```diff
@@ -0,0 +1,38 @@
+package io.realm.entities;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import io.realm.RealmBaseRecyclerAdapter;
+import io.realm.RealmResults;
+
+public class RealmRecyclerAdapter extends RealmBaseRecyclerAdapter<AllTypes, RealmRecyclerAdapter.ViewHolder> {
```

Comment:
Please add a detailed header describing what this class does and any current limitations, e.g. currently none of the `ItemRange*` callbacks are supported. See e.g. https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/java/io/realm/RealmBaseAdapter.java#L24-Lundefined